src/components/StructuredData.tsx

```
@@ -0,0 +1,38 @@
+      "Progress Tracking",
+      "Focus Sessions"
+    ],
+    "softwareVersion": "1.0.0",
+    "datePublished": "2024-01-01",
+    "inLanguage": "en-US"
+  };
+
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)